actors/stage/src/main/java/com/ea/orbit/actors/Stage.java

```diff
@@ -77,6 +79,9 @@
     @Wired
     Container container;
 
+    @Inject
```

Comment:
I think the metrics manager approach actually makes more sense in this context. It means it's usable from any Orbit module. 
Extension also doesn't make a lot of sense as this metrics need to be integrated into the core to be effective.